airflow-e2e-tests/docker/docker-compose-with-localstack.yaml

```diff
@@ -0,0 +1,388 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think we should rather use our docker compose quick start and extend it with local stack. Docker compose is "composable" you can take several docker compose files and docker-compose will merge them together, so it would be enough here to have just a "diff" and apply it to the docker-compose we already have and publish in docs.